python/ray/tests/test_tempfile.py

```diff
@@ -74,14 +74,15 @@ def test_raylet_tempfiles(shutdown_only):
     assert top_levels.issuperset({"sockets", "logs"})
     log_files = set(os.listdir(node.get_logs_dir_path()))
     log_files_expected = {
-        "log_monitor.out", "log_monitor.err", "plasma_store.out",
-        "plasma_store.err", "monitor.out", "monitor.err", "redis-shard_0.out",
-        "redis-shard_0.err", "redis.out", "redis.err", "raylet.out",
-        "raylet.err", "gcs_server.out", "gcs_server.err"
+        "log_monitor.log", "plasma_store.out", "plasma_store.err",
+        "monitor.log", "redis-shard_0.out", "redis-shard_0.err", "redis.out",
+        "redis.err", "raylet.out", "raylet.err", "gcs_server.out",
+        "gcs_server.err", "dashboard.log", "dashboard_agent.log"
     }
 
     for expected in log_files_expected:
-        assert expected in log_files
+        log_files = set(os.listdir(node.get_logs_dir_path()))
+        wait_for_condition(lambda: expected in log_files)
```

Comment:
I don't think this is correct, but I may be misunderstanding. `wait_for_condition` will wait for expected to be in `log_files`, but the lambda doesn't refresh the `log_files` variable in the loop while it waits. I think that line 84 and line 85 should both be encapsulated in the `wait_for_condition`

Reply:
Ah, you are definitely right. I realized this yesterday after I pushed the fix. I will fix it!